src/Main.java

```diff
@@ -1,5 +1,97 @@
+                "jou keuze is: ");
+        int characterChoice = input.nextInt();
+        System.out.println();
+
+        switch (characterChoice) {
+            case 1:
+                System.out.println(vulcanion.toString());
+                playedCharacter = vulcanion;
+                break;
+            case 2:
+                System.out.println(golduck.toString());
+                playedCharacter = golduck;
+                break;
+            case 3:
+                System.out.println(bulbasaur.toString());
+                playedCharacter = bulbasaur;
+                break;
+            case 4:
+                System.out.println(manectric.toString());
+                playedCharacter = manectric;
+                break;
+            default:
+                // vraag: met welke commando zou ik het programma helemaal kunnen stoppen?
+                System.out.println("ongeldige keuze");
+                break;
+        }
+
+        // vraag: of speler HP punten wil weten van gekozen karakter.
+        System.out.println();
+        System.out.print("Wil je mijn HP punten weten ? (j/n) ");
+        String hpRequest = input.next();
+        if (hpRequest.equalsIgnoreCase("j")) {
+            System.out.println();
+            System.out.println("Ik heb " + playedCharacter.getHp() + " HP punten");
+        } else {
+            System.out.print("");
+            // het lukte mij niet om de code te laten werken zonder deze lege 'else' statement te plaatsen. Kan een 'if' nooit zonder 'else'?
+        }
+
+        // vraag aan speler: Welke Pokemon moet  uitgedaagd worden? Het zou netter en  minder type fout gevoelig zijn geweest om met een keuze menu te werken, zoals hierboven, maar ik wilde ook eens deze manier proberen
+            System.out.println();
+            System.out.print("Welke Pokemon wil je uitdagen? (type hele naam): ");
+            String challengedPokemon = input.next();
+
+            if (challengedPokemon.equalsIgnoreCase("Golduck")) {
+                challengedCharacter = golduck;
+                System.out.print(challengedPokemon + " heeft " + golduck.getHp() + " HP punten. ");
+
+            } else if (challengedPokemon.equalsIgnoreCase("Vulcanion")) {
+                challengedCharacter = vulcanion;
+                System.out.print(challengedPokemon + " heeft " + vulcanion.getHp() + " HP punten. ");
+
+            } else if (challengedPokemon.equalsIgnoreCase("Bulbasaur")) {
+                challengedCharacter = bulbasaur;
+                System.out.print(challengedPokemon + " heeft " + bulbasaur.getHp() + " HP punten. ");
+            } else if (challengedPokemon.equalsIgnoreCase("manectric")) {
+                challengedCharacter = manectric;
+                System.out.print(challengedPokemon + " heeft " + manectric.getHp() + " HP punten. ");
+            } else {
+                System.out.println("speler bestaat niet");
+            }
```

Comment:
Je zou dit code blok nog in een while loop kunnen zetten, zodat als de input niet goed is, de speler het opnieuw kan proberen.